.github/workflows/release.yaml

```diff
@@ -90,13 +90,13 @@ jobs:
           CHART_VERSION="${{ needs.build.outputs.chart_version }}"
           helm package helm
```

Comment:
Should this be
```suggestion
          helm package charts/connaisseur
```
going forward?

Reply:
yes, but i would make this change after we completely switch to the `charts/connaisseur` repo

Reply:
What's the rationale? The resources are already there and all other links point to charts/connaisseur already afaik

Reply:
i don't really care 🤷 changed the references.